src/stores/Annotation/Annotation.js

```diff
@@ -677,14 +677,22 @@ export const Annotation = types
       self.setDraftSelected();
       self.versions.draft = result;
       self.setDraftSaving(true);
-
-      return self.store.submitDraft(self, params).then(self.onDraftSaved);
+      return self.store.submitDraft(self, params).then((res) => {
+        self.onDraftSaved;
```

Comment:
```suggestion
  self.onDraftSaved(res);
```
This was previously getting invoked but now is not.